2.adoc

```diff
@@ -73,4 +73,151 @@ Optional description
 . Minus B
 
 
+=== Sublime Text
+
+* Multiplatformní desktopový editor textových souborů a zdrojových kódů s vysokou mírou přizpůsobení prostřednictvím pluginů vyvíjených komunitou
+* link:+https://www.sublimetext.com/+[https://www.sublimetext.com/]
+
+==== Porovnání rozdílů
+* Nejedná se o webovou komponentu, ale o desktopovou aplikaci
+* Univerzální editor, není omezený pouze na markdown jazyky
+* Defaultně neumí zobrazovat preview markdown dokumentu ani jeho outline (možnost doinstalovat pluginy)
+* Obsahuje propracovanou command pallete, přes kterou lze obsluhovat téměř všechna nastavení
+
+==== Zhodnocení UI
+
+* Základní UI po instalaci
+** Zobrazený je pouze zdrojový kód s podporou zvýrazňování syntaxe markdown (detekuje na základě přípony .md)
+** Podpora pro více tabů
+
```

Comment:
Tady uz to nejde a mluvime jen o markdownu, tak bych to zminil

`NOTE: Protože Sublime Text je generický editor a použitelný jako LML editor pouze prostřednictvím pluginů, rozhodl jsem se prozkoumat pouze jazyk Markdown.`